examples/deadline/All-In-AWS-Infrastructure-SEP/python/package/lib/service_tier.py

```diff
@@ -0,0 +1,182 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import typing
+from dataclasses import dataclass
+
+from aws_cdk.core import (
+    Construct,
+    Duration,
+    Stack,
+    StackProps
+)
+from aws_cdk.aws_ec2 import (
+    BastionHostLinux,
+    BlockDevice,
+    BlockDeviceVolume,
+    IVpc,
+    SecurityGroup,
+    SubnetSelection,
+    SubnetType
+)
+from aws_cdk.aws_elasticloadbalancingv2 import (
+    ApplicationProtocol
+)
+from aws_cdk.aws_iam import (
+    ManagedPolicy,
+    Role,
+    ServicePrincipal 
+)
+from aws_cdk.aws_secretsmanager import (
+    Secret
+)
+from aws_cdk.aws_route53 import (
+    IPrivateHostedZone
+)
+
+from aws_rfdk import (
+    DistinguishedName,
+    IMountableLinuxFilesystem,
+    X509CertificatePem
+)
+from aws_rfdk.deadline import (
+    DatabaseConnection,
+    RenderQueue,
+    RenderQueueHostNameProps,
+    RenderQueueTrafficEncryptionProps,
+    RenderQueueExternalTLSProps,
+    Repository,
+    Stage,
+    ThinkboxDockerRecipes,
+)
+
+
+@dataclass
+class ServiceTierProps(StackProps):
+    """
+    Properties for ServiceTier
+    """
+    # The VPC to deploy service tier resources into.
+    vpc: IVpc
+    # The database to connect to.
+    database: DatabaseConnection
+    # The file system to install Deadline Repository to.
+    file_system: IMountableLinuxFilesystem
+    # The path to the directory where the staged Deadline Docker recipes are.
+    docker_recipes_stage_path: str
+    # Internal DNS zone for the VPC
+    dns_zone: IPrivateHostedZone
+
+
+class ServiceTier(Stack):
+    """
+    The service tier contains all "business-logic" constructs
+    (e.g. Render Queue, UBL Licensing/License Forwarder, etc.).
+    """
+
+    def __init__(self, scope: Construct, stack_id: str, *, props: ServiceTierProps, **kwargs):
+        """
+        Initialize a new instance of ServiceTier
+        :param scope: The scope of this construct.
+        :param stack_id: The ID of this construct.
+        :param props: The properties for this construct.
+        :param kwargs: Any kwargs that need to be passed on to the parent class.
+        """
+        super().__init__(scope, stack_id, **kwargs)
+
+        # A bastion host to connect to the render farm with.
+        # The bastion host is for convenience (e.g. SSH into RenderQueue and WorkerFleet instances).
+        # This is not a critical component of the farm, so can safely be removed.
+        self.bastion = BastionHostLinux(
```

Comment:
imo, it would be better if we strip the example down to just what's needed for the SEP. It makes the delta for adding SEP components clearer. The example can be framed as demonstrating just the SEP parts, and that the user could add the parts to the basic example.

Also reduces the amount of copy/paste that exists between the examples.